### PR TITLE
add example to quote a post with an image

### PR DESCRIPTION
**Pull request by zzstoatzz**
https://bsky.app/profile/alternatebuild.dev/post/3lcjijbyeqn2b

### CODE REVIEW

**Comment by MrGeislinger** [comment]
As the dumb person from the post who needed this help, up voting since this would have helped me out a lot!

**Comment by MarshalX** [comment]
Thank you! I have an idea to extend the existing example: [send_embed.py](https://github.com/MarshalX/atproto/blob/main/examples/advanced_usage/send_embed.py). The existing example shows how to send an external link. Let's expand it to media and record. WDYT?

**Comment by zzstoatzz** [comment]
> Thank you! I have an idea to extend the existing example: [send_embed.py](https://github.com/MarshalX/atproto/blob/main/examples/advanced_usage/send_embed.py). The existing example shows how to send an external link. Let's expand it to media and record. WDYT?

Good idea! can do

File: examples/advanced_usage/send_embed.py [hunk 1]
```diff
@@ -1,10 +1,15 @@
+import httpx
 from atproto import Client, models
 
+# image of a condescending cat stare
+DEFAULT_IMAGE_URL = 'https://cdn.bsky.app/img/feed_fullsize/plain/did:plc:xbtmt2zjwlrfegqvch7fboei/bafkreiae4rcokecag5qlpvg6l3otzulqn3hllnazmz2ezyqfl6xzpy5noe@jpeg'
```
Comment (MarshalX) [inline examples/advanced_usage/send_embed.py:5]: pls use image from this account https://bsky.app/profile/test.marshal.dev

Comment (zzstoatzz) [inline examples/advanced_usage/send_embed.py:5]:
```suggestion
DEFAULT_IMAGE_URL = 'https://cdn.bsky.app/img/avatar/plain/did:plc:kvwvcn5iqfooopmyzvb4qzba/bafkreicwqcugdgubtawr6xv6jjifoju67eigwx2xgz4zs73nkzzn36oucy@jpeg'
```

File: examples/advanced_usage/send_embed.py [hunk 2]
```diff
@@ -1,10 +1,15 @@
+import httpx
 from atproto import Client, models
 
+# image of a condescending cat stare
```
Comment (zzstoatzz) [inline examples/advanced_usage/send_embed.py:4]:
```suggestion
```

File: examples/advanced_usage/send_embed.py (hunk 1 above)
Comment (zzstoatzz) [reply examples/advanced_usage/send_embed.py]: @MarshalX done!

image

**Review by MarshalX — APPROVED** [review]

**Comment by MarshalX** [comment]
thank you!

**Pull request merged**